src/cgame/cg_players.cpp

```diff
@@ -91,7 +91,8 @@ sfxHandle_t CG_CustomSound( int clientNum, const char *soundName )
 		}
 	}
 
-	Sys::Drop( "Unknown custom sound: %s", soundName );
+	Log::Warn( "Unknown custom sound: %s", soundName );
+	return ci->sounds[ 0 ];
```

Comment:
What is `ci->sounds[ 0 ]` supposed to be? I would expect it to simply return 0 like for a shader that is not found

Reply:
Apparently, `ci->sounds[ 0 ]` is supposed to be "*death1".